CONTRIBUTING.md

```diff
@@ -3,7 +3,8 @@
 ## Build and deploy
 
 ```bash
+sudo snap install rockcraft --classic --edge
 rockcraft pack -v
-sudo skopeo --insecure-policy copy oci-archive:sdcore-gnbsim_1.4.0_amd64.rock docker-daemon:sdcore-gnbsim:1.4.0
-docker run sdcore-gnbsim:1.4.0
+sudo skopeo --insecure-policy copy oci-archive:sdcore-gnbsim_1.4.3_amd64.rock docker-daemon:sdcore-gnbsim:1.4.3
```

Comment:
```suggestion
sudo rockcraft.skopeo --insecure-policy copy oci-archive:sdcore-gnbsim_1.4.3_amd64.rock docker-daemon:sdcore-gnbsim:1.4.3
```

Reply:
How do I commit this suggestion now? I think I did them in reverse